### PR TITLE
Homework02-exercise06-Yingda Zhao Michael

### PR DESCRIPTION
**Pull request by CodeArt98702**

### CODE REVIEW

File: class02/exercise_6.py [hunk 1]
```diff
@@ -23,7 +23,10 @@ def calculation(a, b):
 	'''
 
 	# code up your solution here
-
+	def calculation(a,b):
+    		sum = a + b
+    		diff = a - b
+    		return [sum,diff];
```
Comment (emma-oc) [inline class02/exercise_6.py:29]: make sure to understand you are returning a list instead of two values

File: class02/exercise_6.py [hunk 2]
```diff
@@ -37,7 +40,9 @@ def triangle_lambda():
 	Returns:
 	lambda_triangle_area: the lambda
 	'''
-
+	def triangle_lambda():
+		lambda_triangle_area = lambda b,h: b*h/2
```
Comment (emma-oc) [inline class02/exercise_6.py:44]: indentation seems to be longer

File: class02/exercise_6.py [hunk 3]
```diff
@@ -82,7 +93,30 @@ def perfect_number():
 	'''
 
 	# code up your answer here
-
-
+	def is_prime_number(x):
```
Comment (emma-oc) [inline class02/exercise_6.py:96]: indentation seems to be  wrong... please fix

File: class02/exercise_6.py [hunk 4]
```diff
@@ -82,7 +93,30 @@ def perfect_number():
...
+    		if x >= 2:
+        		for y in range(2,x):
+           			 if not ( x % y ):
+                			return False
+   	 	else:
+        		return False
+    		return True;
+	
+	def multiplyList(myList) :
```
Comment (emma-oc) [inline class02/exercise_6.py:105]: seems to be doing what `numpy.prod` is supposed to do. check it out: https://docs.scipy.org/doc/numpy/reference/generated/numpy.prod.html

**Comment by emma-oc** [comment]
good job, some minor but important fixes like indentation - can't seem to run code with wrong indentation

File: class02/exercise_6.py (hunk 4 above)
Comment (CodeArt98702) [reply class02/exercise_6.py:105]: This is very helpful. Thank you!

**Pull request closed without merging**